src/constants/label.js

```diff
@@ -1,9 +1,24 @@
 "use strict";
 
 // Name fields in order of preference
-export const name_en = [
-  "coalesce",
-  ["get", "name:en"],
-  ["get", "name:latin"],
-  ["get", "name"],
-];
+export const localizedName = (function () {
+  let userLocales =
+    "languages" in navigator ? [...navigator.languages] : [navigator.language];
```

Comment:
The user-preferred language could be Arabic or Hebrew, which are both written right to left. As tail work, we should install the [mapbox-gl-rtl-text](https://github.com/mapbox/mapbox-gl-rtl-text/) plugin to keep the map readable for users of these languages.

Reply:
#580